ggml.c

```diff
@@ -4905,6 +4911,90 @@ struct ggml_tensor * ggml_flash_ff(
     return result;
 }
 
+// ggml_map_unary
+
+struct ggml_tensor * ggml_map_unary_impl_f32(
```

Comment:
The other `ggml_[...]_impl` functions that already existed don't have static. I was trying to use a similar approach as what already existed rather than making the map ops functions special.

The idea of making those static sounds reasonable, but I think it probably should happen as a separate change where they're all converted at the same time.